src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -1358,9 +1364,22 @@ private void InitializeFromDatabaseRecord(DatabaseRecord record)
             DocumentsStorage.DocumentPut.InitializeFromDatabaseRecord(record);
             ExpiredDocumentsCleaner = ExpiredDocumentsCleaner.LoadConfigurations(this, record, ExpiredDocumentsCleaner);
             TimeSeriesPolicyRunner = TimeSeriesPolicyRunner.LoadConfigurations(this, record, TimeSeriesPolicyRunner);
+            _serverStore.LicenseManager.LicenseChanged += LoadTimeSeriesPolicyRunnerConfigurations;
```

Comment:
How many times this is called? Shouldn't this be registered only once in a lifetime of DocumentDatabase?

Reply:
This worries me with the chance of having a reference held here.

Reply:
Yes, I moved it to `DocumentDatabase.Initialize`